server/routes/pubRedirects.js

```diff
@@ -109,8 +117,8 @@ app.get(
 					: res.redirect(`${prefix}/release/${pubData.releases.length}`);
 			}
 			return versionNumber
-				? res.redirect(`${prefix}/draft/${Number(versionNumber) + 1}`)
-				: res.redirect(`${prefix}/draft`);
+				? res.redirect(`${prefix}/draft/${Number(versionNumber) + 1}${accessHashString}`)
```

Comment:
Depending on the details of what goes into `prefix` you might find the `pubUrl()` util to be a good abstraction here.

Reply:
We don't have an actual community object to pass to `pubUrl()` here. We could query for that, but it felt heavier than I wanted for a redirect route.